ansible_base/rbac/management/commands/RBAC_checks.py

```diff
@@ -0,0 +1,50 @@
+from collections import defaultdict
+
+from django.core.management.base import BaseCommand
+
+from ansible_base.rbac import permission_registry
+from ansible_base.rbac.models import ObjectRole, RoleDefinition
+
+
+class Command(BaseCommand):
```

Comment:
This command doesn't seem to do anything other than output human-oriented text.  Seems like it ought to fail when it detects something wrong or provide some program-consumable output or something.

Or perhaps this shouldn't even be a management command, and should instead be a system check.  https://docs.djangoproject.com/en/5.0/topics/checks/

Reply:
This is too slow for Django checks... way too slow. I always thought this command might outlive its usefulness. I don't want to make the call it delete it until I've gotten more checks done with large data sets, which I've only loaded up in AWX before.

Reply:
Fair enough, but it IMO should have at least a fail / no-fail return code.  Styling would be nice on top of that.